views/index.ejs

```diff
@@ -1,76 +1,264 @@
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="css/index.css">
+
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/fonts.css">
+    <link rel="stylesheet" href="css/index/index.css">
+
+    <script src="scripts/_menu.js" defer></script>
+    <script src="scripts/_language.js" defer></script>
+    <script src="scripts/_variant_font.js" defer></script>
+    <script src="scripts/_accordion.js" defer></script>
+    <script src="scripts/_select.js" defer></script>
 </head>
 <body>
-    <div class="hero-img"></div>
-    
-    <div class="wrapper">
-        <header>
-            <a href="/">
-                <img class="mini-logo" src="assets/mini.png" alt="">
-            </a>
-            <nav>
-                <ul>
-                    <li><a href="/" id="home"><%= __("Home") %></a></li>
-                    <li><a href="/future" id="future-plans"><%= __("Future") %></a></li>
-                    <%- include("partials/language_select.ejs", {path: "/", locale: getLocale()}) %> 
-                </ul>
-            </nav>
+    <div id="select">
+        <div id="popup">
+            <svg id="popup-cancel" viewBox="0 0 24 24">
+                <rect x="9" y="9" width="12" height="1.25" rx=".95" ry=".95"></rect>
+            </svg>
+            <p class="semi-bold"><%= __("test-select-title") %></p>
+            <button id="start-complete-button" class="button primary no-arrow"><%= __("test-select-complete-button") %></button>
+            <p class="question-count"><%= __("test-select-complete-info") %></p>
+            <p><%= __("test-select-divider") %></p>
+            <button id="start-basic-button" class="button secondary no-arrow"><%= __("test-select-basic-button") %></button>
+            <p class="question-count"><%= __("test-select-basic-info") %></p></div>
+        <div id="overlay"></div>
+    </div>
+    <div class="grid-container">
+        <header id="nav">
+            <div id="logo-container">
+                <svg id="home-icon" viewBox="0 0 275 275" fill="var(--clr-text-primary)"
+                     xmlns="http://www.w3.org/2000/svg">
+                    <g id="rim">
+                        <path d="M274.854 137.427C274.854 213.326 213.326 274.854 137.427 274.854C112.43 274.854 88.9926 268.18 68.7981 256.517C65.851 254.815 66.4813 250.524 69.7489 249.573L72.794 248.686C73.8338 248.383 74.9503 248.518 75.8983 249.042C94.1391 259.119 115.113 264.854 137.427 264.854C207.803 264.854 264.854 207.803 264.854 137.427C264.854 115.062 259.092 94.0434 248.973 75.7735C248.447 74.8247 248.312 73.7065 248.615 72.6651L249.5 69.6244C250.451 66.3586 254.739 65.7264 256.443 68.6701C268.152 88.8935 274.854 112.378 274.854 137.427Z"
```

Comment:
Look into importing these from an asset folder instead of pasting the entire SVG to its tag.

Reply:
[Moved home icon svg to external partial home_icon_partial.ejs.](https://github.com/Ankyrian/politivector/pull/6/commits/a6efefe67bee3c53bfe685fd09203bb4ec2291e3)